internal/test/wsu/wsu_test.go

```diff
@@ -106,6 +106,9 @@ func runWSUTestSuite(t *testing.T, vm e2ef.WindowsVM) {
 	cmd := exec.Command("ansible-playbook", "-vvv", "-i", hostFilePath, playbookPath)
 	out, err := cmd.CombinedOutput()
 	require.NoError(t, err, "WSU playbook returned error: %s, with output: %s", err, string(out))
+	if err = framework.WriteToArtifactDir(out, t.Name()+"_"+vm.GetCredentials().GetInstanceId()); err != nil {
+		fmt.Printf("could not write to artifact dir: %s", err)
```

Comment:
Please mention the filename that you failed on in the error message.

Reply:
done